cedar-policy-core/src/evaluator.rs

```diff
@@ -17,8 +17,9 @@
 //! This module contains the Cedar evaluator.
 
 use crate::ast::*;
-use crate::entities::{Dereference, Entities};
+use crate::entities::{Dereference, Entities, EntityAttrValues};
 use crate::extensions::Extensions;
+#[cfg(test)]
 use std::collections::HashMap;
```

Comment:
instead of the `cfg(test)`, could we move this import into the test module(s) where it's needed?

Reply:
It's used in a top level function that is `#[cfg(test)]`, so I'm not sure?

Reply:
Ah got it